opentracing-impl/src/main/java/io/opentracing/NoopTracer.java

```diff
@@ -11,16 +11,13 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.noop;
+package io.opentracing;
 
-import io.opentracing.Span;
-import io.opentracing.Tracer;
-
-public class NoopTracer implements Tracer {
+public final class NoopTracer implements Tracer {
 
     @Override
     public SpanBuilder buildSpan(String operationName) {
-        return NoopSpanBuilder.Instance;
+        return NoopSpanBuilder.INSTANCE;
```

Comment:
I was actually thinking we can get rid of these singletons and use normal member refs:

```java
public NoopTracer() {
    this.span = new NoopSpan(this);
    this.builder = new NoopSpanBuilder(this.span);
}
```